cmd/rac/output.go

```diff
@@ -92,17 +92,25 @@ The following columns parse the values further:
   "15..4" (value in rac 0x4)
   "15..0" which is the full image (value in rac 0x7)
 - NCBIN FPGAColumns
-  The actual number of FPGA Columns (value in rac is the exponent in 2^N)
+  The actual number of FPGA Columns (value in rac is the exponent in 2^N) 
+  (Old naming CCDitem["NCBIN FPGAColumns”]=2**CCDitem["NColBinFPGA"] )

```

Comment:
```suggestion
  The actual number of FPGA Columns (value in rac is the exponent in 2^N) 
  (Old naming `CCDitem["NCBIN FPGAColumns”] = 2**CCDitem["NColBinFPGA"]`)

```
Also, prefer using either `^` or `**` not a mix (the operator doesn't exist in golang, so either will do)